tests/lxc/install-deps/ubuntu_16.04

```diff
@@ -8,4 +8,4 @@ apt-get install python3-pip docker.io -y
 pip3 install importlib-metadata==2.1.0 pytest requests pyyaml sh
 # Attempting to address https://forum.snapcraft.io/t/lxd-refresh-cause-container-socket-error/8698
 # if core is to be installed by microk8s it fails
-snap install core18 | true
```

Comment:
It might be worth completely removing this line and seeing if this problem still exists.

Reply:
I would rather not do this as part of this PR.

The installed `core18` is only unused (when installing `microk8s`, `core20` will be pulled instead). We should not need this line, but I'm pro keeping it.